training/model_management/environments/model-management/context/requirements.txt

```diff
@@ -2,10 +2,6 @@ azure-core=={{latest-pypi-version}}
 azure-ai-ml=={{latest-pypi-version}}
 azure-identity=={{latest-pypi-version}}
 azureml-evaluate-mlflow=={{latest-pypi-version}}
-ffmpeg=={{latest-pypi-version}}
+ffmpeg-python=={{latest-pypi-version}}
 langcodes=={{latest-pypi-version}}
-protobuf==3.20.0
 sentencepiece==0.1.97
```

Comment:
Could you please point me to where this being used in the code?

Reply:
used when loading tokenizer for some of the models (Jean-Baptiste-camembert-ner)